### DISCUSSION TITLE
Wanna renew Github Student Education Pack, but it keep saying "Hi, XXX! You were last verified as a student on . It is not necessary for you to re verify at this time"

### DISCUSSION THREAD

**Discussion by qishenli** [community/community#143149] [GitHub Education]
### Select Topic Area

General

### GitHub Feature Area

GitHub Education

### Body

My student pack is expired on Oct 8, 2024 and I try to renew it by my new school. But it keeps mentioning "Hi, XXX! You were last verified as a student on . It is not necessary for you to re-verify at this time" and I could not renew it.

**Comment by SyedAliJunaidSherazi** [comment]
It sounds like you're having trouble renewing your GitHub Student Education Pack. Since your verification status hasn't expired yet, you won't be prompted to re-verify. To renew your pack, you may need to wait until your current verification period ends.

If your pack has already expired and you're still seeing this message, try reaching out to GitHub Support directly for assistance. They should be able to help you with the renewal process. Good luck :)

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @qishenli! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by KingBoomie** [comment]
I have had a similar problem for the last 2 months. The UI looks weird, if I am actually verified, why does it also say 'expired' on some recent requests. It also confuses client apps. For example, when trying to verify my Jetbrains account through Github, the result there is also in some quantum position of both verified and not: there's a green "You were successfully authenticated with GitHub, which verified that you are a student." and a red "Unfortunately, we're unable to deliver licenses to your country/region owing to existing legal restrictions.". I verified with Jetbrains support that Estonia is not in any way under sanctions or similar restrictions. 

![Screenshot 2024-11-01 093550](https://github.com/user-attachments/assets/2d03c453-d3cc-41cd-9195-25c28eb82afd)

If the solution here is to contact github support, then could anyone actually provide instructions on how todo that? [Github education support](https://support.github.com/contact/education) has a field for `I have found a technical bug` which unfortunately does not allow to submit a support request, but only refers people to post in this discussion forum. The other categories do not apply.
